### ISSUE TITLE
[TW-911] Segmentation fault

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#946]
_John Florian on 2010-11-23T16:59:43Z says:_ 

 This is from task-1.9.3-1.fc13.x86_64, which has been working very well for me, until this little hiccup:

<pre>
$ task+git log pro:aos.herd "Revised connecting message for ssh/pssh subcommands to show the device's ID along with the resolved IP address.  This is most helpful when tartgeting devices using selectors as opposed to a single device ID."
/usr/bin/task+git: line 5: 28924 Segmentation fault      (core dumped) task "$@"
</pre>

And here's my little task+git wrapper:

<pre>
1. !/bin/sh
1.  task wrapper providing integrated git content tracking

task "$@"

cd ~/.task
git ci *.data -F -  &> /dev/null <<EOF
task+git $@
EOF
</pre>

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2010-11-23T16:59:43Z
Modified: 2014-02-09T02:11:51Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2010-11-23T17:06:34Z says:_ 

 Known and fixed in 1.9.4.  The problem is the combination of "log" and "project:...".

Workaround:

<pre>
task add ...
task done <id>
</pre>
